README-ZH_CN.md

```diff
@@ -1,5 +1,7 @@
 # Trilium注意事项
 
+[![English readme](https://img.shields.io/badge/English-white?logo=data:image/svg%2bxml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj48Y2lyY2xlIGN4PSIyNTYiIGN5PSIyNTYiIHI9IjI1NiIgZmlsbD0iI2YwZjBmMCIvPjxnIGZpbGw9IiMwMDUyYjQiPjxwYXRoIGQ9Ik01MyAxMDBjLTIwIDI2LTM1IDU2LTQ0IDg5aDEzM2wtODktODl6TTUwMyAxODljLTktMzMtMjQtNjMtNDQtODlsLTg5IDg5aDEzM3pNOSAzMjNjOSAzMyAyNCA2MyA0NCA4OWw4OS04OUg5ek00MTIgNTNjLTI2LTIwLTU2LTM1LTg5LTQ0djEzM2w4OS04OXpNMTAwIDQ1OWMyNiAyMCA1NiAzNSA4OSA0NFYzNzBsLTg5IDg5ek0xODkgOWMtMzMgOS02MyAyNC04OSA0NGw4OSA4OVY5ek0zMjMgNTAzYzMzLTkgNjMtMjQgODktNDRsLTg5LTg5djEzM3pNMzcwIDMyM2w4OSA4OWMyMC0yNiAzNS01NiA0NC04OUgzNzB6Ii8+PC9nPjxnIGZpbGw9IiNkODAwMjciPjxwYXRoIGQ9Ik01MTAgMjIzSDI4OVYyYTI1OSAyNTkgMCAwMC02NiAwdjIyMUgyYTI1OSAyNTkgMCAwMDAgNjZoMjIxdjIyMWEyNTkgMjU5IDAgMDA2NiAwVjI4OWgyMjFhMjU5IDI1OSAwIDAwMC02NnoiLz48cGF0aCBkPSJNMzIzIDMyM2wxMTQgMTE0IDE1LTE2LTk4LTk4aC0zMXpNMTg5IDMyM0w3NSA0MzdsMTYgMTUgOTgtOTh2LTMxek0xODkgMTg5TDc1IDc1IDYwIDkxbDk4IDk4aDMxek0zMjMgMTg5TDQzNyA3NWwtMTYtMTUtOTggOTh2MzF6Ii8+PC9nPjwvc3ZnPg==)](https://github.com/zadam/trilium/blob/master/README.md) [![Chinese readme](https://img.shields.io/badge/Chinese-white?logo=data:image/svg%2bxml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIGlkPSJMYXllcl8xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHg9IjAiIHk9IjAiIHZpZXdCb3g9Ii00OSAxNDEgNTEyIDUxMiIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PHN0eWxlPi5zdDF7ZmlsbDojZmZkYTQ0fTwvc3R5bGU+PGNpcmNsZSBjeD0iMjA3IiBjeT0iMzk3IiByPSIyNTYiIGZpbGw9IiNkODAwMjciLz48cGF0aCBjbGFzcz0ic3QxIiBkPSJNOTEgMjk3bDIyIDY4aDcybC01OCA0MiAyMiA2OC01OC00Mi01OCA0MiAyMi02OC01Ny00Mmg3MXpNMjU1IDUzOGwtMTctMjEtMjUgOSAxNC0yMi0xNy0yMSAyNiA3IDE1LTIzIDEgMjcgMjYgNy0yNSAxMHpNMjg4IDQ3N2w4LTI2LTIyLTE2aDI3bDgtMjYgOSAyNmgyNmwtMjEgMTYgOSAyNS0yMi0xNnpNMzMzIDMyOWwtMTEgMjQgMTkgMTktMjctNC0xMSAyNC01LTI2LTI3LTQgMjQtMTMtNC0yNiAxOSAxOHpNMjU1IDI1NmwtMiAyNyAyNSAxMC0yNiA2LTIgMjctMTQtMjMtMjYgNyAxNy0yMS0xNC0yMyAyNSAxMHoiLz48L3N2Zz4=)](https://github.com/zadam/trilium/blob/master/README-ZH_CN.md) [![Russian readme](https://img.shields.io/badge/Russian-white?logo=data:image/svg%2bxml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj48Y2lyY2xlIGN4PSIyNTYiIGN5PSIyNTYiIHI9IjI1NiIgZmlsbD0iI2YwZjBmMCIvPjxwYXRoIGQ9Ik00OTYgMzQ1YTI1NSAyNTUgMCAwMDAtMTc4SDE2YTI1NSAyNTUgMCAwMDAgMTc4bDI0MCAyMiAyNDAtMjJ6IiBmaWxsPSIjMDA1MmI0Ii8+PHBhdGggZD0iTTI1NiA1MTJjMTEwIDAgMjA0LTY5IDI0MC0xNjdIMTZjMzYgOTggMTMwIDE2NyAyNDAgMTY3eiIgZmlsbD0iI2Q4MDAyNyIvPjwvc3ZnPg==)](https://github.com/zadam/trilium/blob/master/README.ru.md)
```

Comment:
I'd rather avoid inline images in the README. IMHO standard hyperlinks should suffice ...